src/bindings/python/tests/test_runtime/test_core.py

```diff
@@ -229,7 +229,7 @@ def test_available_devices(device):
 
 def test_get_property(device):
     core = Core()
-    conf = core.get_property(device, "SUPPORTED_CONFIG_KEYS")
+    conf = core.get_property(device, "SUPPORTED_PROPERTIES")
```

Comment:
can we use `properties.supported_properties()` instead of hardcoded string?
In all other similar places as well